conftest.py

```diff
@@ -0,0 +1,4 @@
+from itemadapter import ItemAdapter
+from zyte_common_items import ZyteItemAdapter
+
+ItemAdapter.ADAPTER_CLASSES.appendleft(ZyteItemAdapter)
```

Comment:
Could you please double-check that this file is not installed alongside with the package?